configs/osm_config.yaml

```diff
@@ -0,0 +1,407 @@
+    VC: "saint-vincent-and-the-grenadines"
+    # SX: "saint-marteen"  # Islands
+    TT: "trinidad-and-tobago"
+    # TC: "turks-and-caicos"  # Islands
+    # UM: "united-states-minor-outlying-islands"  # Islands
+    # VI: "united-states-virgin-islands"  # Islands
+    BZ: "belize"
+    CR: "costa-rica"
+    HN: "honduras"
+    GT: "guatemala"
+    NI: "nicaragua"
+    PA: "panama"
+    SV: "el-salvador"
+
+  SouthAmerica:
+    AR: "argentina"
+    BO: "bolivia"
+    BR: "brazil"
+    CL: "chile"
+    CO: "colombia"
+    EC: "ecuador"
+    FK: "falkland-islands"  # Islands
+    GF: "french-guiana"
+    GY: "guyana"  # No Data
+    PE: "peru"
+    PY: "paraguay"
+    SR: "suriname"
+    UY: "uruguay"
+    VE: "venezuela"
+
+
+
+# Based on: https://waml.org/waml-information-bulletin/46-3/index-to-lc-g-schedule/1-world/
+# Australasia region includes New Caledonia and Papua New Guinea
+continent_regions:
+  SCR: ["DK", "NO", "SE", "FI", "IS"]  # SCANDINAVIAN REGION
+  EER: ["BY", "PL", "CZ", "RU", "SK", "UA", "LT", "LV", "EE", "FI", "MD"]  # EASTERN EUROPEAN REGION
+  CER: ["AT", "CH", "CZ", "DE", "HU", "PL", "SK", "LI"]  # CENTRAL EUROPEAN REGION
+  BPR: ["AL", "BA", "BG", "GR", "HR", "ME", "RO", "SI", "RS", "ME", "MK"]  # BALKAN PENISULAN REGION
+  WER: ["FR", "BE", "GB", "IE", "LU", "MC", "NL", "AD"]  # WESTERN EUROPE
+  SER: ["ES", "AD", "IT", "PT", "SM", "MT"]  # SOUTHERN EUROPEAN REGION
+  NAR: ["EG", "LY", "TN", "DZ", "MA", "EH", "SD", "SS"]  # NORTHERN AFRICAN REGION
+  WAR: ["MR", "ML", "NE", "NG", "BJ", "BF", "TG", "GH", "CI", "LR", "SL", "GN", "SN", "GM"]  # WESTERN AFRICAN REGION
+  CAR: ["TD", "CF", "CM", "GQ", "GA", "CD", "CG", "AO"]  # CENTRAL AFRICAN REGION
+  EAR: ["ER", "ET", "UG", "KE", "RW", "BI", "TZ", "MZ", "DJ", "MG"]  # EASTERN AFRICAN REGION
+  SAR: ["MW", "ZM", "ZW", "BW", "NA", "SZ", "LS", "ZA"]  # SOUTHERN AFRICAN REGION
+  KVR: ["AZ", "GE", "AM"]  # Asian regions
+  WAS: ["TR", "AM", "AZ", "BH", "CY", "GE", "IQ", "IL", "JO", "KW", "LB", "OM", "PS", "QA", "SA", "SY", "AE", "YE"]  # WEST ASIAN REGION
+  FEAR: ["JP", "KP", "KR", "CN", "TW", "MN"]  # FAR EASTERN ASIAN REGION
+  SEAR: ["LA", "TH", "KH", "VN", "PH", "MY", "SG", "BN", "ID"]  # SOUTHEASTERN ASIAN REGION
+  CASR: ["KZ", "KG", "UZ", "TM", "TJ"]  # CENTRAL ASIAN REGION
+  SASR: ["MM", "BD", "BT", "NP", "IN", "LK", "PK", "AF"]  # SOUTHERN ASIAN REGION
+  ASEAN: ["VN", "TH", "ID", "PH", "MY", "MM", "KH", "LA", "SG", "BN"]  # ASEAN
+  MEAR: ["TR", "SY", "LB", "CY", "IQ", "IR", "JO", "IL", "PS", "AE", "YE", "KW", "BH", "QA", "SA", "OM"]  # MIDDLE EASTERN ASIAN REGION
+  NACR: ["CA", "GL", "MX", "US"]  # American continent regions
+  LACR: ["AR", "BO", "BR", "CL", "CO", "EC", "GF", "PE", "PY", "SR", "UY", "VE"]  # SOUTHERN LATIN AMERICAN REGION
+  CACR: ["BZ", "GT", "SV", "HN", "NI", "CR", "PA"]  # CENTRAL AMERICAN REGION
+  AUO: ["AU", "NC", "NZ", "PG"]  # Australasia
+  UnitedNations: "cc.convert(names=cc.UN.name_short, to='ISO2')"  # UnitedNations
```

Comment:
Sorry, noticed now, the value is a python code that shall be substituted with the list of countries. you may locally use a notebook and execute that code to create a list of entries that can be copied then here.

Reply:
done.